server.go

```diff
@@ -46,6 +46,10 @@ type Server struct {
 	// For example, it allows OPTIONS /foo even if it is not declared (only GET /foo is declared).
 	corsMiddleware func(http.Handler) http.Handler
```

Comment:
Do you still need this then?

Reply:
True, as it is not exposed, we can remove it, no need for deprecation notice!